### DISCUSSION TITLE
allow guest access to custom page?

### DISCUSSION THREAD

**Discussion by wivaku** [filamentphp/filament#9828] [Help]
I have Filament panel + resources, which require a logged in user.

I also created a custom Filament page.
E.g.
```php
use Filament\Pages\Page;

class MyCustomPage extends Page
{
    protected static string $view = 'filament.pages.my-custom-page';
    protected static bool $shouldSkipAuthorization = true; // just in case
    //...
}
```

This custom page is shown for logged in users, but would like to also show it to guest users.
Is that possible?

Tried this:
```php
Route::prefix('guest')->group(function () {
    Route::view('/my-custom-page', 'filament.pages.my-custom-page');
});
```

But that results in `Error: using $this when not in object context`

**Comment by coolsam726** [comment]
I don't know if this is currently possible for the main panel layout, the only way of achieving this that I can think about is to let your custom page extend `SimplePage` instead of `Page`. This is the layout that is normally used for the Login and Registration pages and it is a guest layout.
Then you can maybe override $layout in your page and set it to a more customized layout for example to remove the centered card and make it full page.
```php
use Filament\Pages\SimplePage;

class SamplePage extends SimplePage
{
    protected static ?string $navigationIcon = 'heroicon-o-document-text';
    protected static string $view = 'filament.tenant-portal.pages.sample-page';
    protected static string $layout = 'your-customized-layout-here';
}
```
Any further advice on this from anyone will be much appreciated.
